pkgs/tools/X11/xpra/default.nix

```diff
@@ -56,41 +83,81 @@ in buildPythonApplication rec {
       inherit libfakeXinerama;
     })
     ./fix-41106.patch  # https://github.com/NixOS/nixpkgs/issues/41106
+    ./fix-122159.patch # https://github.com/NixOS/nixpkgs/issues/122159
```

Comment:
You didn't commit this file and can we fetchpatch the change?

Reply:
:/ oops. Just pushed it.